primitives/src/const_vec.rs

```diff
@@ -92,6 +92,33 @@ impl<T: IntoSchema> IntoSchema for ConstVec<T> {
     }
 }
 
+impl<T> IntoIterator for ConstVec<T> {
+    type Item = T;
+
+    type IntoIter = <Vec<T> as IntoIterator>::IntoIter;
+
+    fn into_iter(self) -> Self::IntoIter {
+        self.into_vec().into_iter()
+    }
+}
+
+/// Trait to extend `[T]` with a method to convert it to `ConstVec<T>` by analogy with `[T]::to_vec()`.
+pub trait ToConstVecExt {
```

Comment:
```suggestion
pub trait ToConstVec {
```

`Ext` suffix implies that there already exists a trait of the same name

Reply:
I was thinking more about that: https://rust-lang.github.io/rfcs/0445-extension-trait-conventions.html

But yes, my current style follow neither of these, will rename